src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -172,6 +172,28 @@ ov_property_key_enable_profiling;
 OPENVINO_C_VAR(const char*)
 ov_property_key_device_priorities;
 
+/**
+ * @brief Read-write property<string> for setting that enables performance improvement by binding
+ * buffer to hw infer request
+ * @ingroup ov_property_c_api
+ */
+OPENVINO_C_VAR(const char*)
+ov_property_key_intel_auto_device_bind_buffer;
+
+/**
+ * @brief Read-write property<string> to enable/disable CPU as accelerator (or helper device) at the beginning
+ * @ingroup ov_property_c_api
+ */
+OPENVINO_C_VAR(const char*)
+ov_property_key_intel_auto_enable_startup_fallback;
+
+/**
+ * @brief Read-write property<string> to enable/disable runtime fallback to other devices when infer fails
+ * @ingroup ov_property_c_api
+ */
+OPENVINO_C_VAR(const char*)
+ov_property_key_intel_auto_enable_runtime_fallback;
```

Comment:
@riverlijunjie do you think we need a dedicated file for non-common properties? Just not to keep everything is one file

Reply:
yes, I think so, which will make the header file more clear, similar with openvino/c/gpu/gpu_plugin_properties.h @WeldonWangwang do we have chance to create a dedicated file for auto plugin's properties of C header file before FF?

Reply:
Hi @ilya-lavrenov, @riverlijunjie  Added dedicated files for auto plugin.